integrations/event-handler/go.mod

```diff
@@ -154,8 +154,8 @@ require (
 	github.com/go-openapi/jsonreference v0.21.0 // indirect
 	github.com/go-openapi/swag v0.23.0 // indirect
 	github.com/go-piv/piv-go v1.11.0 // indirect
-	github.com/go-webauthn/webauthn v0.11.2 // indirect
-	github.com/go-webauthn/x v0.1.14 // indirect
+	github.com/go-webauthn/webauthn v0.12.1 // indirect
+	github.com/go-webauthn/x v0.1.18 // indirect
```

Comment:
Note that go-webauthn is still being updated here (and in the terraform module).